src/__test__/plugin.test.js

```diff
@@ -13,6 +13,11 @@ import {
 import { Map, List } from "immutable";
 import createMarkdownPlugin from "../";
 
+const applyMDtoInlineStyleChange = editorState =>
```

Comment:
not sure what is better
- copy this function here from  new tests and leave it here until old styled tests will be migrated (will they?)
- simply import it from new tests

Reply:
Simply import it from new tests, old tests will stay as they are I'm pretty sure!